testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -432,7 +432,6 @@ def params_from_base_suite_setup(request):
             log_info("Restarting sync gateway {}".format(sg_ip))
             sg_obj.restart_sync_gateways(cluster_config=cluster_config, url=sg_ip)
             # Giving time to SG to load all docs into it's cache
-            time.sleep(240)
```

Comment:
If you remove this, this will fail when you run with enable sample bucket and run query tests

Reply:
I ran the full functional tests for this and it looks good